src/components/FeatureViews/Scheduler/NotEnrolledStudentDialog.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { withRouter } from "react-router-dom";
+import Button from "@material-ui/core/Button"
+import BackArrow from "@material-ui/icons/ArrowBack";
+
+const NotEnrolledStudentsDialog = (props) => {
+    const [open, setOpen] = React.useState(true);
+
+    const handleClose = () => {
+        props.history.goBack();
+    };
+
+    return (
+        <div>
+            <Dialog
+                open={open}
+                onClose={handleClose}
+                aria-labelledby="alert-dialog-title"
+                aria-describedby="alert-dialog-description"
+            >
+                <DialogTitle id="alert-dialog-title">{"OOPS!"}</DialogTitle>
+                <DialogContent>
+                    <DialogContentText id="alert-dialog-description">
```

Comment:
Add the error_outline icon that's colored as orange to warn that this page doesn't exist. make it big and centered.

Reply:
Is this good? I made the background grey to make it seem more of a pop-up. 

<img width="1792" alt="Screen Shot 2020-04-04 at 7 49 45 PM" src="https://user-images.githubusercontent.com/40410706/78465678-83bf6600-76ad-11ea-8183-2a0c348ad7cb.png">

Heres without : 

<img width="1792" alt="Screen Shot 2020-04-04 at 7 51 36 PM" src="https://user-images.githubusercontent.com/40410706/78465693-b4070480-76ad-11ea-8069-f5b2c840c659.png">

Reply:
sorry, just noticed these comments. Add more padding to the top of the thing and we're good to go

Reply:
Added padding.

Reply:
@jsflca2012  can you show a screenshot?